example/properties_directory/non-scalar/file.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "my_json.json",
+    "type": "text",
+    "objectData": {
+        "CONTAINER": "production-20160630-cluster-001",
```

Comment:
Use `include` statement like in other examples

Reply:
An include statement is now used to reference objectData